Quick/Reporting/XCTestObservationCenter+Reporting.m

```diff
@@ -0,0 +1,135 @@
+//
+//  XCTestObservationCenter+Reporting.m
+//  Quick
+//
+//  Created by Paul Young on 1/2/15.
+//  Copyright (c) 2015 Brian Ivan Gesiak. All rights reserved.
+//
+
+#import "XCTestObservationCenter+Reporting.h"
+#import <XCTest/XCTestRun.h>
+#import "XCTestDriver.h"
+#import <objc/runtime.h>
+
+@implementation XCTestObservationCenter (Reporting)
+
++ (void)load {
+    NSDictionary *environment = [[NSProcessInfo processInfo] environment];
+    NSString *reporter = [environment objectForKey:@"QUICK_REPORTER"];
+    
+    if (!(reporter == nil || [reporter isEqualToString:@"xcode"])) {
```

Comment:
I'm seeing that using `QUICK_REPORTER=nyan` does not work for the iOS test target.